wwwroot/css/profile.css

```
@@ -0,0 +1,212 @@
+/* Profile page styling inspired by provided UI mock.
+   Kept compatible with existing Razor form fields (no backend changes). */
+
+.lp-page-bg {
+  background: #f8f7f5;
+}
+
+.lp-breadcrumb {
+  display: flex;
+  gap: 8px;
+  align-items: center;
+  font-family: "Work Sans", system-ui, -apple-system, Segoe UI, Roboto, Arial, sans-serif;
```

Stylelint (17.4.0)

[error] Expected quotes around "Segoe UI" (font-family-name-quotes)

(font-family-name-quotes)